README.md

```diff
@@ -3,45 +3,92 @@
+
+    $ http://download.redis.io/releases/redis-3.2.6.tar.gz
+    $ tar xzf redis-3.2.6.tar.gz
+    $ pushd redis-3.2.6
+    $ make
+    $ ./src/redis-server --daemonize
+    $ popd
+
+Webdis packages its dependencies with the exception of libevent. It is entirely likely that libevent is already available on your system. If so, the following should work. If libevent is not available, compilation will die quickly. However, libevent is in all the common repositories (e.g., yum, apt, brew, etc), and compiling by source is straight forward. 
+
+    $ git clone https://github.com/nicolasff/webdis.git
+    $ pushd webdis
+    $ make
+    $ ./webdis &
+    $ popd
+
+Last, redbiom itself can be installed as a normal Python package.
+
+    $ git clone https://github.com/wasade/redbiom.git
+    $ pip install -e .    
+
+# Examples
+
+The first example block surrounds loading data, because without anything in the cache, there is nothing fun to do. The second block highlights some example commands that can be run, or chained together, for querying the data loaded.
 
-# Load some stuff
+## Load some data
 
-To make use of this cache, we need to load things. Loading can be done in parallel. First, we'll load up metadata. This will create keys in Redis which describe all of the columns associated with a sample (e.g., `metadata-categories:<sample_id>` and hash buckets for each category and sample combination (e.g., `category:<category_name>` as the hash and `<sample_id>` as the field). 
+To make use of this cache, we need to load things. Loading can be done in parallel. First, we'll load up metadata. This will create keys in Redis which describe all of the columns associated with a sample (e.g., `metadata:categories:<sample_id>`, hash buckets for each category and sample combination (e.g., `metadata:category:<category_name>` as the hash and `<sample_id>` as the field), a set of all known categories (e.g., `metadata:categories-represented`), and a set of all known sample IDs (e.g., `metadata:samples-represented`):
 
-	$ ./redbiom admin load-sample-metadata --metadata path/to/qiime/compat/mapping.txt
+	$ redbiom admin load-sample-metadata --metadata path/to/qiime/compat/mapping.txt
 
-Next, we'll load up associations between every single observation in the BIOM table to all the samples its found in. This will create Redis sets and can be accessed using keys of the form `samples:<observation_id>`. 
+**IMPORTANT**: values containing `/` characters cannot be represented the forward slash is used to denote arguments with Webdis. At present, these values are omitted. This is more generally a problem for dates which have not been normalized into an ISO standard. See issue #9.
 
-	$ ./redbiom admin load-observations --table /path/to/biom/table.biom
+**IMPORTANT**: values which appear to be null are not stored. The set of values currently considered nulls are: 
+    
+    {'Not applicable', 'Unknown', 'Unspecified', 'Missing: Not collected',
```

Comment:
We no longer infer null values in Qiita. In the ideal world, users should be using the EBI standard...